tests/test_definitions/column_aggregate_expectations/expect_column_min_to_be_between.json

```diff
@@ -19,6 +19,15 @@
         "zz": "datetime",
         "a": "int",
         "b": "int"
+      },
```

Comment:
Can you help me understand your strategy for where to include these type declaration blobs? 

Are you relying on the defaults, and only overriding when necessary?

Reply:
(This is the only question that needs an answer before I can sign off.)

Reply:
Correct--and to be slightly more explicit, we're relying on the *pandas* type inference performed in `to_sql` and only overriding when we know that's wrong (but if we override one type (e.g. datetime) we have to override them all).

Fundamentally, this should be informed by a more thoughtful review of how we *want* types to behave relative to the implementation layer, since it's important to have the flexibility of *data first* testing (i.e. the test (set) should be expressible and semantically understandable independent of the specific implementation).